setup.py

```diff
@@ -7,8 +7,8 @@
 PACKAGE_VERSION = '0.1'
 
 deps = [
-    'manifestparser',
-    'marionette-client',
+    'manifestparser == 0.9',
+    'marionette-client == 0.8.6',
```

Comment:
marionette-client depends on manifestparser, you should be able to remove the manifestparser dependency.